src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -678,9 +678,13 @@ public override void VisitParameter(IParameterSymbol symbol)
             // (e.g. field types, param types, etc), which just want the name whereas parameters are
             // used on their own or in the context of methods.
 
-            var includeType = format.ParameterOptions.IncludesOption(SymbolDisplayParameterOptions.IncludeType);
+            // SignatureOnlyParameterSymbol.ContainingSymbol throws.
+            var isFunctionPointerParameter = symbol is not Microsoft.CodeAnalysis.CSharp.Symbols.PublicModel.Symbol { UnderlyingSymbol: SignatureOnlyParameterSymbol }
```

Comment:
>SignatureOnlyParameterSymbol [](start = 136, length = 28)

Could you please elaborate how SignatureOnlyParameterSymbol comes to life here?

Reply:
Yeah, this was the part I was trying to dig into just now. I'm not sure how this is ever true.

Reply:
@AlekseyTs @333fred 

There were a number of test failures till I fixed this.  You can see them at https://dev.azure.com/dnceng/public/_build/results?buildId=994331&view=ms.vss-test-web.build-test-results-tab

Reply:
I see, this has nothing to do with function pointer symbols, it's other scenarios that involve SignatureOnlyParameterSymbols that is being guarded against.